{"schemaVersion":{"major":0,"patch":0,"minor":3},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)"},"primaryContentSections":[{"declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"},{"parameters":[{"content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"The error that caused the authentication failure."}]}],"name":"error"}],"kind":"parameters"}],"abstract":[{"text":"Invoked when a rest call fails for authentication reasons.","type":"text"}],"sections":[],"hierarchy":{"paths":[["doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent"]]},"variants":[{"paths":["\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:s5ErrorP","text":"Error","kind":"typeIdentifier"},{"text":")","kind":"text"}],"symbolKind":"case","title":"ZendeskEvent.authenticationFailed(error:)","externalID":"s:10ZendeskSDK0A5EventO20authenticationFailedyACs5Error_p_tcACmF","roleHeading":"Case","modules":[{"name":"ZendeskSDK"}],"role":"symbol"},"kind":"symbol","references":{"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent/authenticationFailed(error:)":{"type":"topic","title":"ZendeskEvent.authenticationFailed(error:)","kind":"symbol","identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)","abstract":[{"type":"text","text":"Invoked when a rest call fails for authentication reasons."}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"text":")","kind":"text"}],"role":"symbol","url":"\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)"},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK":{"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","kind":"symbol","url":"\/documentation\/zendesksdk","abstract":[],"title":"ZendeskSDK","type":"topic","role":"collection"},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ZendeskEvent","kind":"identifier"}],"title":"ZendeskEvent","abstract":[{"type":"text","text":"The set of events that can be emitted from Zendesk SDK."}],"type":"topic","kind":"symbol","role":"symbol","navigatorTitle":[{"text":"ZendeskEvent","kind":"identifier"}],"url":"\/documentation\/zendesksdk\/zendeskevent","identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent"}}}